lib/entities/organization-space-membership.js

```diff
@@ -0,0 +1,32 @@
+/**
```

Comment:
I think the whole file can also go alway as the entity to be used is Space Membership.

Reply:
🧐hmmm. yeah.. I think so. i will test it and try to remove it.